Technology/Artificial Intelligence/interactive & great resources/README.md

```diff
@@ -0,0 +1,45 @@
+# Interactive Tools and other great Resources
+
+# Interactive Tools
```

Comment:
Add this in AI Readme file(https://github.com/colearninglounge/co-learning-lounge/blob/master/Technology/Artificial%20Intelligence/README.md) with heading "Interactive Tools"